backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -669,7 +669,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
           customPollStatusFailure orElse {
             case (_: ExecutionHandle, exception: Exception) if isFatal(exception) =>
               // Log exceptions and return the original handle to try again.
-              jobLogger.warn(s"Fatal exception polling for status. Job will fail.")
+              jobLogger.warn(s"Fatal exception polling for status. Job will fail.", exception)
```

Comment:
ToL: I assume we only see this if the job is really actually about to fail? I don't want to see 100s of these popping up for retried jobs.

Reply:
Yep